src/vectorchord/advanced-features/rerank-in-table.md

```diff
@@ -0,0 +1,30 @@
+# Rerank In Table
+
+To maximize performance, VectorChord stores both the quantized and original vectors in the index.
```

Comment:
`rerank_in_table` also prevents proper prefetching, making it unsuitable for use as a disk index.